Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Services/UserRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Healthcare_Hospital_Management_System.Models;
+
+namespace Healthcare_Hospital_Management_System.Services
+{
+    public class UserRepository : IUserRepository
+    {
+        readonly Dictionary<string, string> users = new Dictionary<string, string>
+        {
+            { "zinchenko.liudmyla@gmail.com", "8b5b9db0c13db24256c829aa364aa90c6d2eba318b9232a4ab9313b954d3555f" },
```

Comment:
At least change the content of the User Repository, avoid committing user sensitive data

Reply:
Removed personal entries, leave only testcases without personal info.